blocks/alert-bar-block/themes/news.json

```diff
@@ -21,33 +21,37 @@
 						"margin-top": "var(--global-spacing-1)",
 						"filter": "drop-shadow(0px 0px 0px rgba(0, 0, 0, 0.0))"
 					},
-					"link": {
-						"color": "var(--global-white)",
-						"filter": "drop-shadow(0px 0px 0px rgba(0, 0, 0, 0.0))",
-						"font-family": "var(--font-family-primary)",
-						"font-size": "var(--body-font-size-small)",
-						"font-weight": "var(--global-font-weight-7)",
-						"padding-bottom": "var(--global-spacing-5)",
-						"padding-left": "var(--global-spacing-4)",
-						"padding-right": "var(--global-spacing-5)",
-						"padding-top": "var(--global-spacing-5)",
-						"text-decoration": "none"
-					},
-					"link-hover": {
-						"color": "var(--text-color)"
-					},
 					"icon": {
 						"fill": "var(--global-white)",
 						"filter": "drop-shadow(0px 0px 0px rgba(0, 0, 0, 0.0))"
 					}
 				}
+			}
+		}
+	},
+	"alert-bar-link": {
+		"styles": {
+			"default": {
+				"color": "var(--global-white)",
+				"filter": "drop-shadow(0px 0px 0px rgba(0, 0, 0, 0.0))",
+				"font-family": "var(--font-family-primary)",
+				"font-size": "var(--body-font-size-small)",
+				"font-weight": "var(--global-font-weight-7)",
+				"padding-bottom": "var(--global-spacing-5)",
+				"padding-left": "var(--global-spacing-4)",
+				"padding-right": "var(--global-spacing-5)",
+				"padding-top": "var(--global-spacing-5)",
+				"text-decoration": "none"
 			},
 			"desktop": {
-				"components": {
-					"link": {
-						"font-size": "var(--body-font-size)"
-					}
-				}
+				"font-size": "var(--body-font-size)"
+			}
+		}
+	},
+	"alert-bar-link-hover": {
+		"styles": {
+			"default": {
+				"color": "var(--text-color)"
```

Comment:
These styles should still be applied with `"components": { "link": {}, "link-hover": {} }`. To really fix this issue the conflicting uses of `padding` and `padding-*` need to be corrected. We'd need to search though all of the blocks setting padding on the `link` component and make sure they are using the same properties to set those styles. Breaking them out into `padding-top`, `padding-bottom`, `padding-left`, and `padding-right` would be ideal since it will make the conversion to logical properties easier.